contracts/SlashIndicator.sol

```diff
@@ -197,6 +203,15 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function downtimeSlash(address validator, uint256 count) public override {
```

Comment:
No access control for this function?

Reply:
Done

Reply:
`require(msg.sender == block.coinbase || ....`.  Coinbase can slash others at will?

Reply:
Fixed